doc/source/whatsnew/v1.5.0.rst

```diff
@@ -619,6 +619,7 @@ Reshaping
 - Bug in concanenation with ``IntegerDtype``, or ``FloatingDtype`` arrays where the resulting dtype did not mirror the behavior of the non-nullable dtypes (:issue:`46379`)
 - Bug in :func:`concat` with identical key leads to error when indexing :class:`MultiIndex` (:issue:`46519`)
 - Bug in :meth:`DataFrame.join` with a list when using suffixes to join DataFrames with duplicate column names (:issue:`46396`)
+- Bug in :meth:`merge` allows empty suffixes, keeping the columns as-is
```

Comment:
add the issue number here (or this PR number); can you expand this, its not really clear what you are fixing from the note